zeroeq/receiver.cpp

```diff
@@ -180,6 +177,22 @@ Receiver::~Receiver()
     _impl->remove(this);
 }
 
+Receiver::Receiver(Receiver&& from)
+    : _impl(from._impl)
+{
+    from._impl.reset();
+}
+
+Receiver& Receiver::operator=(Receiver&& from)
```

Comment:
Ah yes, didn't think (so far) as you can do it in the definition (and didn't click the link yet). Will try.